app/scripts/controllers/deleteuser.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name bballApp.controller:DeleteuserCtrl
+ * @description
+ * # DeleteuserCtrl
+ * Controller of the bballApp
+ */
+angular.module('bballApp')
+  .controller('DeleteuserCtrl', ['$scope', 'user', '$location', '$rootScope', function ($scope, user, $location, $rootScope) {
+
+    $scope.confirmDelete = function(username) {
+      var promise = user.attemptDelete(username);
+
+      promise.then(function (success) {
+        $location.path('/landing');
+        $rootScope.userName = null;
```

Comment:
Should avoid defining anything on the $rootScope as we want to avoid global objects if possible. Place it in a service instead.